pkgs/servers/mastodon/default.nix

```
@@ -7,6 +7,7 @@
 , version ? import ./version.nix
 , srcOverride ? null
 , dependenciesDir ? ./.  # Should contain gemset.nix, yarn.nix and package.json.
+, yarnSha256 ? null
```

```suggestion
, yarnSha256Override ? null
```
? Because it is only needed when using `srcOverride`?